docs/design/libraries/DllImportGenerator/StructMarshalling.md

```diff
@@ -221,6 +209,32 @@ To enable blittable generics support in this struct marshalling model, we extend
 
 Since all fields typed with non-parameterized types are validated to be blittable at type definition time, we know that they are all blittable at type usage time. So, we only need to validate that the generic fields are instantiated with blittable types.
 
+#### Design 2: `DisableRuntimeMarshallingAttribute`
+
+As an alternative design, we can solve the `unmanaged` vs `blittable` dichotomy and the ref assembly problem by changing the definition of `blittable` to be the same as `unmanaged`. The `DisableRuntimeMarshallingAttribute` attribute helps us solve this problem. When applied to an assembly, this attribute causes the definition of `blittable` to change to "`unmanaged` types with no auto-layout fields" for all P/Invokes in the assembly, among other features. This definition will work for 99% of our scenarios.
```

Comment:
> by changing the definition of `blittable` to be the same as `unmanaged`.

Should we just consider the blittable term defunct? In the sense blittable still means what it did prior to .NET 7 and we focus on adopting what "unmanaged" means and applying? My concern is that if blittable now means something slightly different then there may be confusion around Interop because it will mean different things for older runtimes. I think this is mainly around how we doc and talk to the community.

> This definition will work for 99% of our scenarios.

I would mention at least one case where it doesn't. I assume the sole case is below. I'd make that clear.